pandas/core/groupby.py

```diff
@@ -1826,6 +1826,55 @@ def pipe(self, func, *args, **kwargs):
         """
         return _pipe(self, func, *args, **kwargs)
 
+    @Substitution(name='groupby')
+    @Appender(_doc_template)
+    def is_monotonic_increasing(self):
+        """
+        Returns whether each group is monotonically increasing.
+
+        Equivalent to ``.apply(lambda x: x.is_monotonic_increasing)``.
+
```

Comment:
Can you add a `.. versionadded:: 0.22.0` line here (and in the other as well)?

I also think we should keep this for 0.22.0, as it is a new feature, 0.21.1 is for bug fixes.

Reply:
@jorisvandenbossche thanks for the feedback. I've gone ahead and fixed the whatsnew references and added the ``.. versionadded:: 0.22.0`` on my local branch.

I wanted to clarify the `_series_apply_whitelist` thing. It appears there's a long list of methods in `_common_apply_whitelist` - you'd want me to add the new methods there? And would there need to be any other changes?

Reply:
I think you need to add it to `_series_apply_whitelist`, as it is only a Series method, not a DataFrame method. If you add that, you can remove the `def is_monotonic_..` functions/